src/Kiota.Builder/Writers/Go/CodeMethodWriter.cs

```diff
@@ -424,13 +425,20 @@ private void WriteRequestBuilderBody(CodeClass parentClass, CodeMethod codeEleme
                                                 .ToList();
             var skipIndex = requestParams.requestBody == null ? 1 : 0;
             requestInfoParameters.AddRange(paramsList.Where(x => x == null).Skip(skipIndex).Select(x => "nil"));
+            
             var paramsCall = requestInfoParameters.Any() ? requestInfoParameters.Aggregate((x,y) => $"{x}, {y}") : string.Empty;
+
+            if (!string.IsNullOrEmpty(defaultParams))
+                paramsCall = $"{defaultParams} {paramsCall}";
+            
             writer.WriteLine(template(generatorMethodName, paramsCall));
         }
-        private static void WriteExecutorMethodCall(CodeMethod codeElement, RequestParams requestParams, LanguageWriter writer) {
+        private static void WriteExecutorMethodCall(CodeMethod codeElement, RequestParams requestParams, LanguageWriter writer)
+        {
```

Comment:
```suggestion
```

Reply:
just for clarity, remove this method all together since we don't have overloads anymore for the executor method and replace the call above by a invalid operation exception